miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/userinfo/UserInfoHandler.kt

```diff
@@ -0,0 +1,76 @@
+package com.rakuten.tech.mobile.miniapp.userinfo
```

Comment:
Shouldn't this class be under `js` instead since it's closely coupled to `MiniAppMessageBridge`? Maybe it can go under `js/userinfo` if you think there will be more classes related to user in the future.